interface/patient_file/summary/add_edit_issue.php

```diff
@@ -731,15 +733,15 @@ function divclick(cb, divid) {
                     </div>
                 </div>
                 <div class="form-group">
-                    <label class="col-form-label col-2" for="form_begin"><?php echo xlt('Begin Date'); ?>:</label>
+                    <label class="col-form-label col-4" for="form_begin"><?php echo xlt('Begin Date'); ?>:</label>
                     <div class="col-10">
                         <input type='text' class='datepicker form-control' style="width:50%" name='form_begin' id='form_begin'
                         value='<?php echo attr(oeFormatShortDate($irow['begdate'])) ?>'
                         title='<?php echo xla('yyyy-mm-dd date of onset, surgery or start of medication'); ?>'>
                     </div>
                 </div>
                 <div class="form-group" id='row_enddate'>
-                    <label class="col-form-label col-2" for="form_begin"><?php echo xlt('End Date'); ?>:</label>
+                    <label class="col-form-label col-4" for="form_begin"><?php echo xlt('End Date'); ?>:</label>
```

Comment:
This is actually incorrect. The Bootstrap 4 grid system goes all the way up to col-12, not col-14.